tools/serverpod_cli/test/integration/util/model_helper/model_file_load_test.dart

```diff
@@ -0,0 +1,823 @@
+import 'dart:io';
+
+import 'package:path/path.dart';
+import 'package:serverpod_cli/analyzer.dart';
+import 'package:serverpod_cli/src/util/model_helper.dart';
+import 'package:test/test.dart';
+import 'package:uuid/uuid.dart';
+
+GeneratorConfig createGeneratorConfig([
+  List<String> serverPackageDirectoryPathParts = const [],
+]) {
+  return GeneratorConfig(
+    name: 'test',
+    type: PackageType.server,
+    serverPackage: 'test_server',
+    dartClientPackage: 'test_client',
+    dartClientDependsOnServiceClient: true,
+    serverPackageDirectoryPathParts: serverPackageDirectoryPathParts,
+    relativeDartClientPackagePathParts: [],
+    modules: [],
+    extraClasses: [],
+    enabledFeatures: [],
+  );
+}
```

Comment:
Suggestion: Change to a builder, I believe there should be one.

Reply:
Fixed! 👍